kernel/arch/Makefile

```
@@ -6,12 +6,12 @@ include $(TOPDIR)/misc/mk/config.mk
 ifdef ARCH
 SUBDIRS:=$(ARCH)
 else
-SUBDIRS:=ppc sparc x86
+SUBDIRS:=arm ppc sparc x86
```

checkmake (0.2.2)

[warning] Target "SUBDIRS" should be declared PHONY.

(phonydeclared)